relay/kusama/src/xcm_config.rs

```diff
@@ -190,6 +190,7 @@ pub struct XcmConfig;
 impl xcm_executor::Config for XcmConfig {
 	type RuntimeCall = RuntimeCall;
 	type XcmSender = XcmRouter;
+	type XcmRecorder = XcmPallet; // FAIL-CI @cisco please check
```

Comment:
```suggestion
	type XcmRecorder = ();
```

Just set it off for now, it will be enabled in a subsequent PR when dry-run runtime APIs will be added (not useful until then)